students/M1lya1/apple_shop/scripts.js

```diff
@@ -0,0 +1,1324 @@
+ const items = [
```

Comment:
Please, consider to exclude this data to external file. And then import it here.